Services/ADT/classes/Bridges/class.ilADTFormBridge.php

```diff
@@ -200,7 +200,7 @@ abstract public function addToForm() : void;
      * Add ADT-specific JS-files to template
      * @param ilGlobalTemplate $a_tpl
      */
-    public function addJS(ilGlobalTemplate $a_tpl) : void
+    public function addJS(ilGlobalTemplateInterface $a_tpl) : void
```

Comment:
This breaks the unit tests.
Please can you modify class.ilADTGroupFormBridge::addJs, too.